wordpress-mobile/WordPress-iOS
# Remove WordpressComApi class and related services and tests

Refs #5245 

To test:
 - Compile the app
 - Check if no errors and warning shows up regarding WordPressComApi
 - Run the unit tests
 - Run the app and check if all main section are working ok.

Needs review: @koke
